src/marvin/__init__.py

```diff
@@ -4,8 +4,8 @@
 import nest_asyncio as _nest_asyncio
 import asyncio as _asyncio
 
-
-_nest_asyncio.apply()
+loop = _asyncio.get_event_loop_policy().new_event_loop()
```

Comment:
Minor preference for making this private (_loop) to avoid polluting the main namespace

Reply:
good call